doc/modules/classes.rst

```diff
@@ -609,6 +609,25 @@ Kernels:
    isotonic.check_increasing
    isotonic.isotonic_regression
 
+.. _impute_ref:
```

Comment:
Is this referenced anywhere?

Reply:
I am not familiar with reST, I am not sure why there's `_ref` here. This was with other modules here so I used this :( .

Reply:
you need this only if it is referenced somewhere. But I guess if it is not referenced now, it can be in the futur so it can stay here